client/my-sites/importer/importer-blogger.jsx

```diff
@@ -0,0 +1,58 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FileImporter from './file-importer';
+
+class ImporterBlogger extends React.PureComponent {
+	static displayName = 'ImporterBlogger';
+
+	static propTypes = {
+		site: PropTypes.shape( {
+			title: PropTypes.string.isRequired,
+		} ).isRequired,
+		importerStatus: PropTypes.shape( {
+			filename: PropTypes.string,
+			importerState: PropTypes.string.isRequired,
+			errorData: PropTypes.shape( {
+				type: PropTypes.string.isRequired,
+				description: PropTypes.string.isRequired,
+			} ),
+			percentComplete: PropTypes.number,
+			siteTitle: PropTypes.string.isRequired,
+			statusMessage: PropTypes.string,
+		} ),
+	};
+
+	importerData = {
+		title: 'Blogger.com',
+		icon: 'blogger-alt',
+		description: this.props.translate(
+			'Import posts, pages, comments, tags, and images from a Blogger.com export file.'
+		),
+		uploadDescription: this.props.translate(
+			'Upload a {{b}}Blogger.com export file{{/b}} to start importing into {{b}}%(title)s{{/b}}.',
```

Comment:
Would it make sense to make "Blogger.com" a variable so that the string can be reused for other importers? In the end we don't want translators to translate it anyway, right?

Reply:
Yep - that makes sense.

Reply:
I've incorporated that change into this commit - 078c1c9

Reply:
Thanks!